ocs_ci/framework/main.py

```diff
@@ -69,6 +69,8 @@ def init_ocsci_conf(arguments=None):
     args, unknown = parser.parse_known_args(args=arguments)
     ocs_version = args.ocs_version
     load_config(args.ocsci_conf)
+    if ocs_version:
+        framework.config.ENV_DATA["ocs_version"] = ocs_version
```

Comment:
We are loading the ocs version file on line 85 (87 with this change) which has the ocs_version set under ENV_DATA already. Do we just need to move that code higher up here?

Reply:
correct. I didn't see ocs_version in config file.